composer.json

```diff
@@ -65,7 +65,7 @@
         "mockery/mockery": "^1.0",
         "nunomaduro/collision": "^3.0",
         "phpunit/phpunit": "^7.5",
```

Comment:
a little bit to much lines deleted ;-)
compare with master branch

Reply:
hahaha, yuup, sorry about that.

Reply:
Solved the issue, thanks a lot for your time SourceDoctor. Sorry about that.